test/stability/testMerge/testMerge.go

```diff
@@ -230,7 +227,7 @@ func composeStatistics() error {
 	// Test is passed, if p1 and p2 do not differ too much: |p1-p2| < 4%
 	// and enough packets received back
 	if atomic.LoadInt32(&passed) != 0 &&
-		(p1-p2 < 4 || p2-p1 < 4) && received*100/sent > passedLimit {
+		(p1-p2 < 4 && p1-p2 > -4) && received*100/sent > passedLimit {
```

Comment:
I think it was more simple and obvious before...

Reply:
Maybe, unfortunately it was mathematical incorrect, because "|x|<y" is logical AND, not OR